frontend/src/metabase/visualizations/visualizations/WaterfallChart/WaterfallChart.tsx

```diff
@@ -66,7 +67,8 @@ Object.assign(
         // eslint-disable-next-line ttag/no-module-declaration -- see metabase#55045
         props: { title: t`Total color` },
         widget: "color",
-        getDefault: () => color("text-dark"),
+        // Unfortunately, to get static viz to look right, we need to avoid using alpha colors here
+        getDefault: () => Color(color("text-dark", staticVizOverrides)).hex(),
```

Comment:
😬 this feels dirty. 

When we do our follow up, we should unify how we export static viz colors from the colors definition file.

Reply:
Agreed